libs/ui/elements/src/lib/downloads-list/downloads-list.component.ts

```diff
@@ -33,12 +33,37 @@ export class DownloadsListComponent {
 
   activeFilterFormats: FilterFormat[] = ['all']
 
+  private filterByProtocol(
+    links: DatasetOnlineResource[]
+  ): DatasetOnlineResource[] {
+    const preferredLinks = new Map<string, DatasetOnlineResource>()
+    const OGC_FEATURES_PROTOCOL = 'ogcFeatures'
```

Comment:
I don't think we need a const here, using the string literal value should be enough :)